accounts/abi/abi.go

```diff
@@ -309,3 +309,13 @@ func UnpackRevert(data []byte) (string, error) {
 		return "", errors.New("invalid data for unpacking")
 	}
 }
+
+func PackRevert(revertMessage string) []byte {
```

Comment:
It seems you are only using this as a test utility. Please move to the test file directly.

Reply:
Indeed, it's only used in the test. 

Putting it next to `UnpackRevert` seemed like the right place 😄 happy to inline it.

Reply:
https://github.com/ethereum/go-ethereum/pull/30695/commits/b304b1325c22c763bff43bd26836e45545d9dc16